obsproject/obs-studio
# obs-outputs: Drop unused config file

### Description

The obs-outputs-config.h.in file is not actually used. It only defines the FTL_FOUND variable, but the build system takes care of that by setting a compiler argument for that.

Drop that file.

### Motivation and Context

Less is more.

### How Has This Been Tested?

 * Build and run this branch
 * Check that FTL still works as expected

### Types of changes





 - Code cleanup (non-breaking change which makes code smaller or more readable)



### Checklist:
